src/test/java/org/javarosa/core/model/SelectChoiceTest.java

```diff
@@ -111,7 +114,7 @@ public void getChild_returnsEmptyString_whenChoicesAreFromSecondaryInstance_andR
                         t("data id='select-empty'",
                             t("select"))),
                     instance("choices",
-                        t("item", t("label", "Item"), t("property", ""))
+                        t("item", t("label", "Item"), t("name", "item"), t("property", ""))
```

Comment:
This was a mistake introduced because of the lack of binding check for internal secondary instances!